### ISSUE TITLE
Intermediate asset inside the new Asset Browser Inspector

### ISSUE THREAD

**Issue by lsemp3d** [o3de/o3de#12857]
It was discovered the need to include intermediate asset inside Asset Browser. It was discussed that we can postpone this requirement until we develop the new Asset Browser Inspector. The below image is just an example of something that we can include this feature. Discussion on how this will actually operate should be discussed further when the ticket is picked up by a developer. Adding this ticket to Phase 2

 

 

!dependent assets.png|width=651,height=366!

**Comment by rainbj** [comment]
@lsemp3d we might be able to get away with pushing this ticket to phase 3. Take a look and let me know what you think.

**Comment by MiKom** [comment]
IIUC this task depends on [#12872](https://github.com/o3de/o3de/issues/12872) which is in Phase 4. Shouldn't one of them get moved?
